### ISSUE TITLE
Dataset, Dataloader 직관성 문제

### ISSUE THREAD

**Issue by Lee-Gihun** [Lee-Gihun/Temp-Example#1]
train, valid, test 를 정의 하면서 validation set 을 train 으로 분류하는 코드도 있고
test로 분류하는 경우도 있어서 직관성을 해치는 것 같습니다.
